ext/mysqli/mysqli_driver.c

```diff
@@ -28,55 +28,55 @@
 
 
 /* {{{ property driver_report_read */
-static int driver_reconnect_read(mysqli_object *obj, zval *retval, bool quiet)
+static zend_result driver_reconnect_read(mysqli_object *obj, zval *retval, bool quiet)
 {
 	ZVAL_BOOL(retval, MyG(reconnect));
 	return SUCCESS;
 }
 /* }}} */
 
 /* {{{ property driver_report_write */
-static int driver_reconnect_write(mysqli_object *obj, zval *value)
+static void driver_reconnect_write(mysqli_object *obj, zval *value)
 {
 	MyG(reconnect) = Z_LVAL_P(value) > 0;
 	return SUCCESS;
 }
```

Comment:
If `void` you need to remove the return statement.

Reply:
Ok, thanks. I see a problem with this though. Even though the property is only accepting long I can set it to any value I want and the outcome is just garbage. Is there a way to restrict the type of value that can be assigned to the property?